packages/social-previews/src/facebook-preview/full-preview.tsx

```diff
@@ -0,0 +1,61 @@
+import { ExternalLink } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import SectionHeading from '../shared/section-heading';
+import FacebookDefaultLinkPreview from './default-link-preview';
+import FacebookDefaultPostPreview from './default-post-preview';
+import FacebookLinkPreview from './link-preview';
+import FacebookPostPreview from './post-preview';
+import type { FacebookPreviewProps } from './types';
+
+import './style.scss';
+
+const FacebookFullPreview: React.FC< FacebookPreviewProps > = ( props ) => {
```

Comment:
Maybe we can come up with a better name that represents what the component does. The component actually displays multiple previews, not just a single preview type. Some names I can think of:
- `FacebookPreviews`
- `FacebookPreviewsList`
- `FacebookPreviewTypes`
- `FacebookPostPreviews`
etc.

Reply:
Yeah, sounds good.